src/js/components/Distribution/stories/typescript/Simple.tsx

```diff
@@ -1,33 +0,0 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import isChromatic from 'chromatic/isChromatic';
-
-import { Grommet, Box, Distribution, Text } from 'grommet';
-import { grommet } from 'grommet/themes';
-
-const SimpleDistribution = () => (
```

Comment:
Let's remove the js file instead.

Reply:
Removed `Simple.js` instead of `Simple.tsx`